easybuild/easyblocks/p/python.py

```diff
@@ -232,6 +250,10 @@ def configure_step(self):
             }
             self.cfg.update('configopts', "--with-tcltk-libs='%s'" % tcltk_libs)
 
+        # don't add user site directory to sys.path (equivalent to python -s)
+        # This matters e.g. when python installs the bundled pip & setuptools (for >= 3.4)
+        env.setvar('PYTHONNOUSERSITE', '1', verbose=False)
+
```

Comment:
I have several dependencies of EasyBuild in my user site directory. Forcing the installation of `ensurepip` should be more reliable than completely disabling the user's site.

Reply:
Do you have any reference on how the above is "forcing"? For me it wasn't iirc, but might have missed something. 

We do this for all pip packages and it didn't cause any problems yet, did it? So imo this is very reliable already and eb dependencies should not be affected as this env var only takes effect for new processes

Reply:
Yep, `PYTHONNOUSERSITE` is needed. I thought that `--with-ensurepip=install` would force the installation but it's not the case.